### DISCUSSION TITLE
잡음제거 -- blur

### DISCUSSION THREAD

**Discussion by yibnn** [kikiru328/Bone_Detection#43] [Study]
https://deep-learning-study.tistory.com/164

받은 코드에선 미디언필터를 사용했는데 보니까 양방향 필터가 조금 더 결과가 깔끔하게 나올 수도 있을것 같다는 생각이 듭니다.

한번씩 확인해주시고 의견 공유해주세요!!

**Comment by Jjongu** [comment]
미디언필터 사용해도 잡음이 심할때 사용하면 좋을거같습니다
